gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageFileSystem.java

```diff
@@ -44,13 +44,15 @@ public interface GoogleCloudStorageFileSystem {
   URI GCS_ROOT = URI.create(SCHEME + ":/");
 
   /**
-   * Creates and opens an object for writing. If the object already exists, it is deleted.
+   * Creates and opens an object for writing. If the object already exists, it is overwritten.
    *
    * @param path Object full path of the form gs://bucket/object-path.
    * @return A channel for writing to the given object.
    */
   default WritableByteChannel create(URI path) throws IOException {
-    return create(path, CreateFileOptions.DEFAULT_OVERWRITE);
+    return create(
+        path,
+        CreateFileOptions.builder().setWriteMode(CreateFileOptions.WriteMode.OVERWRITE).build());
```

Comment:
nit: why not create a constant DEFAULT_OVERWRITE in CreateFileOptions, rather than creating this every time?

Reply:
It's a pre-mature optimization to have many default values - new Java object allocation/creation should be insignificant in relation to the GCS IO operations that executed after this.